### PR TITLE
Send page layout update.

### LINKED ISSUE #215: update send page layout
use figma!

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #215

image

image

### CODE REVIEW

**Review by JustinBeBoy — CHANGES_REQUESTED** [review]
1. Seem like you update segmented affected other page

image

2. if Fee rate support only BTC I think you can hidden this part
image


1 bold text, 2 "Destination Account", 3,4 the gap is too large
image


I can't select the account has DCR
image


when typing amount, the address text box autofocus
https://github.com/crypto-power/cryptopower/assets/19331824/30b347a0-828b-4b22-91d3-cc17309d07b5

"add recipient" button not work
image

File: ui/page/components/fee_rate_selector.go [hunk 1]
```diff
@@ -112,141 +109,85 @@ func (fs *FeeRateSelector) isFeerateAPIApproved() bool {
...
+								card.Radius = cryptomaterial.CornerRadius{TopRight: 8, TopLeft: 0, BottomRight: 8, BottomLeft: 0}
+								return card.Layout(gtx, fs.SaveRate.Layout)
+							}),
+						)
+					}
+					if fs.feeRateSwitch.SelectedSegment() == values.String(values.StrFetched) {
+						fs.fetchedRatesDropDown.Width = unit.Dp(gtx.Constraints.Max.X) / 2
+						layoutBody = fs.fetchedRatesDropDown.Layout
+					}
+					if !fs.isFeerateAPIApproved() || fs.selectedWalletType() == libutils.LTCWalletAsset || fs.selectedWalletType() == libutils.DCRWalletAsset {
```
[the remark this answers is not in the document]
Comment (dreacot) [reply ui/page/components/fee_rate_selector.go:149]: btc and ltc, it should not sow in the UI for dcr

**Comment by dreacot** [comment]
> "add recipient" button not work

@Sirmorrison even though we are not adding the functionality via this PR, the UI should work

File: ui/page/components/fee_rate_selector.go [hunk 2]
```diff
@@ -112,141 +109,85 @@ func (fs *FeeRateSelector) isFeerateAPIApproved() bool {
...
+					if !fs.isFeerateAPIApproved() {
+						lbl.Text = values.StringF(values.StrNotAllowed, values.String(values.StrFeeRates))
+						lbl.Color = fs.Theme.Color.Danger
+					} else if fs.selectedWalletType() == libutils.LTCWalletAsset || fs.selectedWalletType() == libutils.DCRWalletAsset {
+						// TODO: Add fee rate API query for LTC
+						lbl.Text = values.StringF(values.StrNotSupported, values.String(values.StrFeeRateAPI))
+						lbl.Color = fs.Theme.Color.Danger
+					}
+
+					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
```
[the remark this answers is not in the document]
Comment (Sirmorrison) [reply ui/page/components/fee_rate_selector.go:187]: yes we do for this particular use case.

**Comment by Sirmorrison** [comment]
`I can't select the account has DCR`
Kindly confirm this is not the current behavior on master reason is i know you are not allowed to send funds directly to a mixed account.

**Comment by Sirmorrison** [comment]
> > "add recipient" button not work
> 
> @Sirmorrison even though we are not adding the functionality via this PR, the UI should work

A follow up PR will implement this functionality.. in the mean time, i will just hide this button to prevent any confusion.

File: ui/page/components/fee_rate_selector.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (JustinBeBoy) [reply ui/page/components/fee_rate_selector.go:186]: it causes confusion, you can try the following code

```
gtx.Constraints.Min.X = gtx.Constraints.Max.X
return layout.Inset{Top: values.MarginPadding4}.Layout(gtx, func(gtx C) D {
	return layout.E.Layout(gtx, lbl.Layout)
})
```

**Review by JustinBeBoy — DISMISSED** [review]
1. it still unresolved
b1

2 Please update `coin selection` layout
b2

3. dropdown on fee rates need update
b3

**Comment by Sirmorrison** [comment]
image

There are two different designs.. one has `destination amount` and amount has `Amount`.. i am using `Amount` because `Destination Amount` does not make a sense. There is no field titled `Source Amount` so i dont see why we are have `Destination AMount`

**Comment by Sirmorrison** [comment]
image

I dont think resolving issues with the dropdown is ideal in the PR.. for starters its already large enough.. any other UI updated that does not affect functionality will be addressed ina. follow up PR.

**Comment by JustinBeBoy** [comment]
> image
> There are two different designs.. one has `destination amount` and the other has `Amount`.. i am using `Amount` because `Destination Amount` does not make a sense. There is no field titled `Source Amount` so i dont see why we are have `Destination AMount`

It seems you have made a mistake here, please carefully check the photo I sent, there is only 1 design, not 2, the layout I am talking about here is sent to "wallet", not "address" and you are comparing these two with each other

**Comment by JustinBeBoy** [comment]
> image
> 
> I dont think resolving issues with the dropdown is ideal in the PR.. for starters its already large enough.. any other UI updated that does not affect functionality will be addressed ina. follow up PR.

I don't think it's too difficult, dropdown is available now, simply fix the way it displays and it's in this PR, I think
@dreacot  what do you think ?

**Comment by dreacot** [comment]
> > image
> > I dont think resolving issues with the dropdown is ideal in the PR.. for starters its already large enough.. any other UI updated that does not affect functionality will be addressed ina. follow up PR.
> 
> I don't think it's too difficult, dropdown is available now, simply fix the way it displays and it's in this PR, I think @dreacot what do you think ?

i see no reason why it should not be resolved here, there a drodowns in master that work as intended already

**Comment by dreacot** [comment]
i believe this part should be sticky

Screenshot from 2023-12-18 14-22-32

**Review by dreacot — APPROVED** [review]

**Pull request merged**